torch/_export/serde/schema.py

```diff
@@ -250,6 +250,7 @@ class GraphModule:
     # TODO(zhxchen17) Merge call_spec into call graph.
     call_spec: CallSpec
     module_call_graph: List[ModuleCallEntry]
+    constants: str
```

Comment:
Do we need to bump schema version?

Reply:
I was thinking no because we haven't published things yet :P but I would bump it after today.

Reply:
OK so you're going to store attributes that are tensors here. Didn't we have a general place to store tensors by reference in the IR as well?